tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -4,7 +4,7 @@
 import types
 import numpy as np
 Tensor = Any
-
+JaxPrecisionType = Any
```

Comment:
Would it be better to bind this to the JAX Precision Object Class? i.e. `<class 'jaxlib.xla_extension.PrecisionConfig_Precision'>`

Reply:
Good point! I'll move the binding into the various generators that generate the jitted functions. We're trying to avoid importing `jax` at global scope so that users don't need to install `jax`.